package.json

```diff
@@ -64,6 +70,7 @@
     "webpack-node-externals": "^1.2.0"
   },
   "dependencies": {
+    "bluebird": "https://registry.npmjs.org/bluebird/-/bluebird-3.5.0.tgz",
```

Comment:
Hmm what's up with the URL here? Is this normal?

Reply:
This is weird. This is what showed up with `npm install --save bluebird` though so I'm not sure why it's different than other dependencies.

Reply:
Yeah... when I just did `npm uninstall --save bluebird` `npm install --save bluebird` it showed up normally so maybe just try doing that?